configurations/Rainier 1S4U Chassis.json

```diff
@@ -17,7 +17,8 @@
             "InputVoltage": [
                 110,
                 220
-            ]
+            ],
+            "PowerConfigFullLoad": true
         },
         {
             "Name": "1600W IBMCFFPS Configuration",
```

Comment:
Did I misunderstand something?
If you have two 1600 watt power supplies in a Rainier 1S4U, then you should not have a oversubscription, so wouldn't two be a full load, and thus true?

Reply:
Okay, clarification in Slack.

We are setting a GPIO to true (1) or false(0) to indicate if we expect a full load (4) or not (2). In configurations that only need 2 supplies of a type, the GPIO gets set to false(0), indicating we only expect the lower amount.